packages/components/src/ListingDetailPhaseCard/ChallengePhaseDetail.tsx

```diff
@@ -10,26 +10,35 @@ export const ChallengePhaseDetail: React.StatelessComponent<ChallengePhaseProps>
     <>
       <MetaRow>
         <MetaItem>
-          <MetaItemLabel>{props.isViewingUserChallenger ? "You are the challenger" : "Challenger"}</MetaItemLabel>
-          <MetaItemValueEthAddress>{getFormattedEthAddress(props.challenger!)}</MetaItemValueEthAddress>
+          <MetaItemLabel>
+            Reward Pool
+            <QuestionToolTip explainerText={<RewardPoolToolTipText />} positionBottom={true} />
+          </MetaItemLabel>
+          <MetaItemValue>{props.rewardPool}</MetaItemValue>
         </MetaItem>
       </MetaRow>
       <MetaRow>
         <MetaItem>
           <MetaItemLabel>
-            Reward Pool
-            <QuestionToolTip explainerText={<RewardPoolToolTipText />} positionBottom={true} />
+            Challenger Deposit
```

Comment:
I believe the Reward Pool is dispPct * deposit, isn't it?

Reply:
You're right. I just updated the tooltip to show the correct dispensationPct from Parameters.